Cargo.toml

```diff
@@ -50,3 +50,8 @@ harness = false
 [[test]]
 name = "test-ptymaster-drop"
 path = "test/test_ptymaster_drop.rs"
+
+[[test]]
```

Comment:
Turns out the test I added in a recent commit essentially does the same thing (pass 2 fds using 1 cmsg), so I'll remove this one.

Adding examples to nix is a very good idea, but I guess it should be done as a more coordinated approach rather than in this PR.